CMake/resolve_dependency_modules/folly/CMakeLists.txt

```diff
@@ -27,40 +27,31 @@ message(STATUS "Building Folly from source")
 
 if(gflags_SOURCE STREQUAL "BUNDLED")
   set(glog_patch && git apply ${CMAKE_CURRENT_LIST_DIR}/folly-gflags-glog.patch)
+  # Together with the patch applied above prevents folly from test compiling a
+  # snippet to find the right namespace (which would fail because gflags isn't
+  # built yet)
+  set(FOLLY_UNUSUAL_GFLAGS_NAMESPACE OFF)
```

Comment:
curious why not build gflags before folly then ?

Reply:
All of this finishes in the configure step, nothing is built only cmake is run BUT the test compilation happens in folly's cmake. So we have to work around this.